etc/gcerc

```diff
@@ -85,6 +85,12 @@ alias gcescp="scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
 # conda.
 alias use.conda3="namemunge PATH ~/opt/conda3/bin"
 alias use.conda2="namemunge PATH ~/opt/conda2/bin"
+# aliases are not expanded when the shell is not interactive.  This cause error
+# message "use.conda3: command not found" dumped to stdout, and may result in
+# the other issues. For example, when non-interactive ssh connection is
+# established to scale up parallel computing, the shell may access the message
+# from the stdout.  Use shopt to always enable aliases expanded.
+shopt -s expand_aliases
 if [ -d ~/opt/conda3/bin ]; then use.conda3; fi
```

Comment:
Agree. Let's use the explicit command instead of aliases (and enabled expand_aliases). I will push a updated PR soon.